src/Device/deviceLister.ts

```diff
@@ -143,28 +158,23 @@ export const stopWatchingDevices = () => {
  *
  * @param {string} serialNumber of the device expected to appear
  * @param {number} [timeout] Timeout, in milliseconds, to wait for device enumeration
- * @param {DeviceListing} [expectedTraits] The traits that the device is expected to have
+ * @param {DeviceTraits} [expectedTraits] The traits that the device is expected to have
  * @returns {Promise} resolved to the expected device
  */
 export const waitForDevice = (
     serialNumber: string,
     timeout = DEFAULT_DEVICE_WAIT_TIME,
-    expectedTraits: DeviceListing = { serialPort: true }
+    expectedTraits: DeviceTraits = DEFAULT_TRAITS
 ) => {
     logger.debug(`Will wait for device ${serialNumber}`);
 
     // To be compatible with `serialport`
-    expectedTraits = {
-        serialPort: expectedTraits.serialport,
-    };
```

Comment:
I am uncertain about this change, because it really looks like the new code does something else than the old code and my understanding of the PR was, that you did not want to change behaviour. 

But while the old code replaced what was in `expectedTraits` with a new object with just the property `serialPort` in it, the new code adds the property `serialPort` to the object, but also leaves all existing properties there. 

Is this correct?